mephisto/scripts/local_db/gh_actions/auto_generate_blueprint.py

```diff
@@ -49,6 +48,7 @@ def main():
         file_name="../../../../docs/web/docs/reference/blueprints.md",
     )
     blueprint_file.new_header(level=1, title="Blueprints")
+    blueprint_file.new_paragraph("The blueprint determines the task content.")
```

Comment:
https://mephisto.ai/docs/explanations/architecture_overview/#blueprints

> The blueprint contains all of the related code required to set up a task run

Reply:
Yeah, I'll do that. I was just copying the output when you run `mephisto wut`